syntax = "proto3";

option csharp_namespace = "IntelliCook.RecipeSearch.Client";

enum RecipeNutritionValue {
    RECIPE_NUTRITION_VALUE_HIGH = 0;
    RECIPE_NUTRITION_VALUE_MEDIUM = 1;
    RECIPE_NUTRITION_VALUE_LOW = 2;
    RECIPE_NUTRITION_VALUE_NONE = 3;
}

message RecipeNutrition {
    RecipeNutritionValue calories = 1;
    RecipeNutritionValue fat = 2;
    RecipeNutritionValue protein = 3;
    RecipeNutritionValue carbs = 4;
}